server/service/src/sync/remote_data_synchroniser.rs

```diff
@@ -314,11 +313,6 @@ impl<'a> RemoteSyncState<'a> {
             .set_bool(KeyValueType::RemoteSyncInitilisationFinished, Some(true))
     }
 
-    pub fn set_site_id(&self, site_id: i32) -> Result<(), RepositoryError> {
```

Comment:
Saved in bootstrap already.

Reply:
Sorry should have noticed earlier but how should this work when providing a config, i.e. if you don't enter bootstrap mode?